stable_baselines/common/base_class.py

```diff
@@ -153,6 +155,49 @@ def _setup_learn(self, seed):
         if seed is not None:
             set_global_seeds(seed)
 
+    @abstractmethod
+    def _get_parameter_list(self):
```

Comment:
quick question: why not a public method? Because it is only used internally?

Reply:
Yes. The intuition was that you could save/load parameters without fiddling with tensorflow stuff. Granted, there shouldn't be anything wrong with accessing this function.